nodejs/awsx/apigateway/README.md

```diff
@@ -202,6 +202,242 @@ let endpoint = new awsx.apigateway.API("example", {
 
 Request body validation is currently not supported. If you have a strong use case, please comment on this [open issue](https://github.com/pulumi/pulumi-awsx/issues/198).
 
+### Lambda Authorizers
+
+[Lambda Authorizers](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html) are AWS Lambda functions that provide control access to an API. You can define a Lambda Authorizer for an Event Handler Route or a Static Route. API Gateway supports `request` or `token` type Lambda authorizers. A `token` Lambda Authorizer uses an authorization token (i.e. a header in the form `Authorization: Token <token>`) to authorize the user, whereas a `request` Lambda Authorizer uses the request parameters (i.e. headers, path parameter or query parameters).
+
+To define an Authorizer, you provide a Lambda that fulfills `aws.lambda.EventHandler<AuthorizerEvent, AuthorizerResponse>` or you provide information on a pre-existing Lambda authorizer. The example below shows defining the Authorizer Lambda directly inline. See the [Event Handler Route](#Event-Handler-Route) section for other ways you can define a Lambda for the Authorizer.
+
+#### Token Authorizer
+
+Below is an example of a custom `request` Lambda Authorizer that uses `awsx.apigateway.getRequestLambdaAuthorizerDefinition` to simplify defining the authorizer.
```

Comment:
consider naming to `getRequestLambdaAuthorizer` as that seems sufficiently clear as to waht it does. This also fits the shape better as you have: `authorizers: [awsx.apigateway.getRequestLambdaAuthorizerDefinition({`.  If it were `authorizerDefinitions: ...` then `getRequestLambdaAuthorizerDefinition` would fit better.  But since you're calling these `authorizers` it makes sense to have the naming on the other side match as well.